subscription-engine/kafka-ws/app/Kafka/KafkaTopicsConsumerCallbacksMgmt.js

```diff
@@ -17,6 +18,7 @@ class KafkaTopicsConsumerCallbacksMgmt {
     // only one callback by topic
     this.topic = new Map();
     this.kafka = new KafkaWSConsumers();
+    StateManager.addHealthChecker('kafka', this.kafka.healthChecker.bind(this.kafka), 5000);
```

Comment:
`StateManager.addHealthChecker('kafka', this.kafka.healthChecker.bind(this.kafka), 5000);`

Can this statement be moved to the `init()` method?